src/main/markdown/release-notes.md

```diff
@@ -59,10 +59,71 @@ The GWT Release Notes
 
 <a id="Release_Notes_Current"></a>
 
-<!-- ## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+- Support for several Java 8 API added.
+- Fix memory leak with Java 8 compilation.
+- Source level set to Java 8 by default
 
 ### Deprecations
-- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'. -->
+- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'.
+- Deprecate c.g.g.validation
```

Comment:
Yeah the reasoning should be added as the code itself generally works fine. If the code will be moved into its own project in the future it should also be mentioned here.